Detectors/TRD/reconstruction/macros/checkTrackletCharges.C

```diff
@@ -23,9 +23,9 @@ using namespace o2::trd;
 std::vector<TH1F*> createTrdChargeHists()
 {
   std::vector<TH1F*> hCharges;
-  hCharges.push_back(new TH1F("Charge0", "Charge window 0;charge;counts", 64, 0, 63));
+  hCharges.push_back(new TH1F("Charge0", "Charge window 0;charge;counts", 256, 0, 255));
   hCharges.push_back(new TH1F("Charge1", "Charge window 1;charge;counts", 256, 0, 255));
-  hCharges.push_back(new TH1F("Charge2", "Charge window 2;charge;counts", 256, 0, 255));
+  hCharges.push_back(new TH1F("Charge2", "Charge window 2;charge;counts", 64, 0, 63));
```

Comment:
Would it make more sense to also set the ranges for Q0/1 to their respective range i.e. 0...127?

`
 hCharges.push_back(new TH1F("Charge0", "Charge window 0;charge;counts", 127, 0, 127));
 hCharges.push_back(new TH1F("Charge1", "Charge window 1;charge;counts", 127, 0, 127));
 hCharges.push_back(new TH1F("Charge2", "Charge window 2;charge;counts", 63, 0, 63));
`

Reply:
I thought that as well, but then its hard to see the spike on the right.

Reply:
Makes sense, the same argument would then hold for Q2 since we also see a spike there.